meson_options.txt

```diff
@@ -86,6 +86,12 @@ option(
 	value: true,
 	description: 'Use the BMD bootloader (not always applicable)'
 )
+option(
+	'stlink_v2_isol',
+	type: 'boolean',
+	value: false,
+	description: 'Build for the ISOL variant of STlink v2'
```

Comment:
For consistency with naming, please use `ST-Link v2` to refer to the family of boards.